nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/purple.scss

```diff
@@ -126,15 +38,14 @@ $material-theme-light: mat.define-light-theme(
                 accent: $material-accent-light,
                 warn: $warn-light
             ),
-            //typography: mat.define-typography-config(), // TODO: typography
-            density: -3
+            density: -1
```

Comment:
I think this density needs to be `-3`. We have double scrollbars on the processor configuration dialog's properties tab without it.
<img width="776" alt="Screenshot 2024-04-09 at 9 10 43 AM" src="https://github.com/apache/nifi/assets/713866/82250e58-33fa-455e-bcba-0da4b4eea524">

Reply:
UPDATE: ok so our dialogs have set sizes (for example this 'Edit Processor' dialog is a a medium dialog size with minWidth: 760px and maxWidth: 760px) and all of our buttons and tables set their own density. So it seems this setting is not such a global setting and it is not as easy as adjusting it to adjust the entire density of the app. I will revert it to `-3` for now and maybe in the future we can support adjusting the density globally via the theme.